test/resources/invalid/typecheck/InvalidRedefinedVar.flo

```diff
@@ -0,0 +1,12 @@
+action GET_ALL_USERS = network(url = '/users', method = 'GET');
+action UPDATE_USER = network(url = '/users/:uuid', method = 'PUT');
+action ENOUGH_USERS_ONLINE = toggle();
+
+flow USERS = {
+    array users = [];
+    int users = 0;
```

Comment:
I know this is an invalid test anyway, but `int` should be `number`.

Reply:
Whoopsies, fixed